test/integration/compare_comply.test.js

```diff
@@ -6,7 +6,7 @@ const authHelper = require('../resources/auth_helper.js');
 const auth = authHelper.auth;
 const describe = authHelper.describe; // this runs describe.skip if there is no auth.js file :)
 
-const compare_comply = new CompareComply(auth.compare_comply);
+const compare_comply = auth ? new CompareComply(auth.compare_comply) : null;
```

Comment:
Instead of doing this, will you wrap all of the integration tests in a `describe('compare_comply_integration'...` block like we do in the other tests? Not sure why this one got left out originally but I would prefer all of the services to be consistent

Reply:
That makes sense and is definitely probably cleaner. Done.

Reply:
Great, thanks!